### ISSUE TITLE
It downloads some empty subtitles

### ISSUE THREAD

**Issue by shaynefortier** [crunchy-labs/crunchy-cli#398]
**Describe the bug**
When you download certain series that have true closed captions like *The Wrong Way to Use Healing Magic* and *Frieren: Beyond Journey's End*, it downloads 2 en-US  (CC) which one is empty and the other contains subtitles.

**To Reproduce**
Steps / command to reproduce the behavior:
```
$ crunchy archive -a ja-JP -a en-US -s en-US -s fr-FR -m audio --include-chapters https://www.crunchyroll.com/series/GG5H5XQX4/frieren-beyond-journeys-end
```

You open a video player such as VLC and under `Subtitle > Sub Track` we see two *en-US (CC) - [English]*

To even make sure that one of those English CC is empty, use the ffprobe command you will get the metadata for the empty subtitle has a duration of 0.
```
...
Stream #0:8(en-US): Subtitle: ass (forced)
    Metadata:
      title           : en-US (CC)
      ENCODER         : Lavc60.31.102 ssa
      DURATION        : 00:00:00.000000000
...
```

**Expected behavior**
There shouldn't be two CC subtitles with one of them empty or both contains

**Screenshots**
Showing VLC found two English streams for English dub
![Screenshot_20240429_062230](https://github.com/crunchy-labs/crunchy-cli/assets/109255560/4efc30dc-01a0-41a6-802a-c61726af4dbe)
Crunchyroll has two English subtitles/Closed Captions for the dub version
![Screenshot_20240429_062343](https://github.com/crunchy-labs/crunchy-cli/assets/109255560/21514427-bbe1-472a-903e-14002c0d6327)
Open the inspector and we see that it loaded a VTT file containing the closed captions.
![Screenshot_20240429_062513](https://github.com/crunchy-labs/crunchy-cli/assets/109255560/f2495174-26fe-4d36-b5c8-ad03023f94a8)

**Client (please complete the following information):**
 - OS: EndevourOS (Arch Linux based)
 - Version 3.5.2 (74aaed 2024-04-24)

**Additional context**
I found that Crunchyroll started to add true Closed Captions in English to certain anime series such as Frieren, but the format is not in plain text nor ASS, but in VTT which we can see in the last screenshot. The issue it cause is when I need to seperate all the streams so that I can add a subtitle language that Crunchyroll doesn't provid, I can't merge them back together using ffmpeg, because the files are corrupted with a text or binary codec stream that ffmpeg doesn't support. And I can't get rid of that codec in the mp4 video even if I re-encode it.

I tested on other series like Goblin Slayer Season 1 I downloaded yesterday and there is no issue, but it doesn't have the VTT closed captions. This is why I think the issue I am having comes from that empty subtitle.

The solution may be to download it in the right codec format (VTT) or check the duration that returns 0 and ignore it.

**Comment by bytedream** [comment]
The subtitles were already downloaded as A `.ass` and `.vtt` (hence the two `CC` subtitle streams). But FFmpeg had problems with the subtitles being in different formats, internally converted vtt to ass, but this didn't went well and removed all dialogs. It got fixed in [v3.6.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.6.0)

**Comment by shaynefortier** [comment]
Thank you! It worked well.
